README.md

```diff
@@ -9,28 +9,83 @@ Run `make test` to run all the unit tests. Run `make test-integration` to run th
+```
 
 ### Configuration
 
 | name                       | description                                | required | default value |
 |----------------------------|--------------------------------------------|----------|---------------|
-| `destination_config_param` | Description of `destination_config_param`. | true     | 1000          |
+| `nodes` | Comma separated list of Cassandra nodes' addresses (at least one), ex: `127.0.0.1:9042`,`127.0.0.2:8080`. | true     |          |
+| `keyspace` | The keyspace name that has the table (similar to a database in a relational database system). | true     |          |
+| `table` | The table name to write data into. | true     |          |
+| `auth.mechanism` | Authentication mechanism used by Cassandra, use `basic` for password auth, and `none` if auth is off. | false     | `none`         |
+| `auth.basic.username` | Username, required only if `basic` auth mechanism is used. | false     |          |
+| `auth.basic.password` | Password, required only if `basic` auth mechanism is used. | false     |          |
+
+### Table name
+If a record contains a `cassandra.table` property in its metadata it will be inserted in that table, otherwise it will 
+fall back to use the table configured in the connector. Thus, a Destination can support multiple tables in a single 
+connector.
+
+## Example pipeline configuration file
+```yaml
+   pipelines:
+   cassandra-pipeline:
+     status: running
+     name: example-pipeline
+     description: write data into Cassandra.
+     connectors:
+       postgres-con:
+         type: source
+         plugin: builtin:postgres # you can use any other source connector, this is just an example.
+         name: postgres-source
+         settings:
+           url: postgresql://username:pass@127.0.0.1:5432/mydb #postgresql://{username}:{password}@{host}:{port}/{database}
+           table: employees
+           orderingColumn: id
+       cassandra-con:
+         type: destination
+         plugin: standalone:cassandra
+         name: cassandra-dest
+         settings:
+           nodes: 127.0.0.1:9042 #{host}:{port}
+           keyspace: company
+           table: employees
+     processors:
+       proc1:
+         type: parsejsonpayload #postgres creates raw data payload, but json formatted, so this processor will convert the raw data into structured. 
+```
+Build your Cassandra connector, then place the connector binary in the `connectors` directory relative to Conduit,
+check [connectors](https://github.com/ConduitIO/conduit#connectors) for more details. Also, check [Pipeline Configuration Files Docs](https://github.com/ConduitIO/conduit/blob/main/docs/pipeline_configuration_files.md)
+ for more details about how to run this pipeline.
 
 ## Known Issues & Limitations
-* Known issue A
-* Limitation A
+* Supports only structured data format key and payload. If your data is raw and json formatted, then you can use
```

Comment:
Maybe we can link to https://github.com/ConduitIO/conduit-site/pull/30 once it's merged?

Reply:
yes sounds good!